corona-warn-app/cwa-app-android
# Update TAN Submission Flow (EXPOSUREAPP-6595)

This PR updates the Submission Flow when using a TAN to allow the new Deletion Warning Screen.

To test this PR:
- Validate that the submission flow for registering a test using a TAN is working correctly (with no test of same type)
- Validate that the Deletion Warning Screen is shown and the flow is correct when using a TAN (with test of same type already registered using TAN or QRCode) 
- Also validate when using a TAN, that after the test registration the `NoConsent` screen is shown and that the submission of the consent is working correctly (when using a TAN in general, consent to submit the keys is not granted before  registration, but afterwards) 

Please note: Comments will be addressed in a polishing PR
